install/kubernetes/Makefile

```
@@ -66,6 +66,7 @@ lint:
 		|| (echo -e "$$crd not found in $(CHART_FILE).\nPlease update the chart to include $$crd."; exit 1); \
 	done
 	$(QUIET)$(HELM) lint --with-subcharts --values ./cilium/values.yaml ./cilium
+	go run ../../tools/chart-checker --chart-path ./cilium
```

Nit: when specifying the full package path, Go will run the local version of a command when run from a directory under a go.mod. This is equivalent and a little clearer:
```suggestion
	go run github.com/cilium/cilium/tools/chart-checker --chart-path ./cilium
```